docs/CHANGELOG.rst

```diff
@@ -3,6 +3,11 @@ Version history
 
 We follow `Semantic Versions <https://semver.org/>`_.
 
+0.2.0
```

Comment:
Usually with saritasa invocations i do it like this
```suggestion
unreleased
```
Then when it's decided to release a new version I change it to new version num and update it pyproject